data/weapons.txt

```diff
@@ -834,8 +834,8 @@ outfit "Torpedo Launcher"
 		"turn" 1.5
 		"homing" 3
 		"optical tracking" .8
-		"shield damage" 680
-		"hull damage" 680
+		"shield damage" 720
+		"hull damage" 720
```

Comment:
I feel like DPS is not the correct unit to focus on when we're talking about slow-to-fire missiles. Damage per missile is the relevant stat since there are so few of them that focus can be paid to individually dodging or shooting them down. As such, I don't tend to take DPS into account for large missiles because I don't feel like it's relevant.